docs/patch.md

```diff
@@ -123,6 +123,39 @@ Here's a minimal example showing how to clear a given path:
 patch.clear('custom.pets');
 ```
 
+### remove
+
+This method removes all values that match in a collection.
+
+The following table shows how the operation behaves in different scenarios:
+
+Current Value                     | Given Value       | Result
+----------------------------------|-------------------|-------------------
+`['a', 'b', 'c']`                 | `'a'`             | `['b', 'c']`
+`['a', 'b', 'c', 'c']`            | `'c'`             | `['a', 'b']`
+`['a', 'b', 'c']`                 | `'d'`               | `['a', 'b', 'c']`
+`[]`                              | `'foo'`           | `[]`
+`['foo', {'qux': 'quux'}, 'bar']` | `{'qux': 'quux'}` | `['foo', 'bar']`
```

Comment:
Since we only use numbers and a, b, c, etc, I think it's better to make this one like this:

`['a', {b: 1}, 'c']` | `{b: 1}` | `['a', 'c']`